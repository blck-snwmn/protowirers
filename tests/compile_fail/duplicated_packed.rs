use protowirers::*;

#[derive(Proto)]
struct Sample {
    #[def(field_num = 1, def_type = "int32", packed, packed)]
    s: u32,
}

fn main() {}
