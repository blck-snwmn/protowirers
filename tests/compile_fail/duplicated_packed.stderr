error: packed is duplicated in #[def(...)].
 --> $DIR/duplicated_packed.rs:5:54
  |
5 |     #[def(field_num = 1, def_type = "int32", packed, packed)]
  |                                                      ^^^^^^
